infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -193,4 +194,11 @@ class LocationTrackController(
         )
         return locationTrackService.getMetadataSections(id, publishType, boundingBox)
     }
+
```

Comment:
SwitchStructurelle ja SwitchOwnerille on luotu omat servicensä ja daonsa niin tässä voisi noudattaa samaa kaavaa eli LocationTrackOwnersin jutut omiin tiedostoihinsa.

Reply:
Vastamielipide: Controller-taso voi mun mielestäni olla ihan `LocationTrackController`:in alla, kun ownerit liittyy aika vahvasti sijaintiraiteisiin. Hyvä idea service- ja dao-tasojen osalta kumminkin.

Reply:
Jätin vielä toistaiseksi LocationTrackin alle. Mietin tuossa että pitäisiköhän common-skeeman jutuille olla oma servicensä eli olisi vaan joku OwnerService, jolta voisi kysellä sijaintiraiteen tai vaihteen omistajatietoja.
Tuntui hieman mälsältä luoda serviceä/daota, jossa olisi vaan yks rivi "getLocationTrackOwners"

Reply:
Olikos tässä muuten selkeää että vaihteen omistajat on eri joukko kuin raiteen? Äkkiseltään tuntuisi että voisi olla vain TrackInfraOwner tms ja molemmat viitata samoihin optiohin. Jos se on kuitenkin selvästi eri joukko niin hyvä näin.